A copy edit review

RGalluzzo opened the pull request:
Introduction General:
Switching between subject, subscriber, and claimant without clearly
aligning with the graphic. I did not change since it wasn’t clear which
was preferred. Suggest updating to be more consistent as part of the
next revision.
Section 1.2:
For example, in order to maintain integrity of the service, it may be
desirable to know the home ZIP Code of a user for purposes of census
taking or petitioning an elected official but where it is not necessary
or desirable to know the underlying identity of the person. – Awkward
example. Suggest shortening.
Section 2.2:
Real-live v. Real-world. They are used interchangeably. I changed all to
real-life for consistency.
The term validation is not used at all in IAL2 or IAL3. There is also no
discussion of evidence in either of these descriptions. Not sure if this
is intentional.
In the discussion of IAL 2&3 and anonymity there is the following
statement: “It may be beneficial to the user that uses a high assurance
CSP for transactions at lower assurance levels.  For example, a user may
maintain an IAL3 identity, yet should be able to use their CSP for IAL2
and IAL1 transactions.” This does not explain a benefit. I provided some
suggested re-wording.
Section 4:
We state that there is only one outcome of identity proofing here, but
in section one we list 4 outcomes. We should update and harmonize the
language in one of the two sections.
4.1.1.6 Address Confirmation
The term verified and validated seem to be used interchangeably with
regard to the address of record matching what is in records. I am not
sure if this is intentional or if one is preferred.
4.4.1.8 Security controls
Very confusing control. It says they shall have moderate or high and
then says they should have moderate.
Section 5
The table for identity evidence uses a wide array of terms for the
person being proofed – from applicant, to subject, to person, to
individual. I am not sure if this is intentional.

Left off at Table 5-2

RGalluzzo committed:
A copy edit review - batch #1

Introduction General:
Switching between subject, subscriber, and claimant without clearly
aligning with the graphic. I did not change since it wasn’t clear which
was preferred. Suggest updating to be more consistent as part of the
next revision.
Section 1.2:
For example, in order to maintain integrity of the service, it may be
desirable to know the home ZIP Code of a user for purposes of census
taking or petitioning an elected official but where it is not necessary
or desirable to know the underlying identity of the person. – Awkward
example. Suggest shortening.
Section 2.2:
Real-live v. Real-world. They are used interchangeably. I changed all to
real-life for consistency.
The term validation is not used at all in IAL2 or IAL3. There is also no
discussion of evidence in either of these descriptions. Not sure if this
is intentional.
In the discussion of IAL 2&3 and anonymity there is the following
statement: “It may be beneficial to the user that uses a high assurance
CSP for transactions at lower assurance levels.  For example, a user may
maintain an IAL3 identity, yet should be able to use their CSP for IAL2
and IAL1 transactions.” This does not explain a benefit. I provided some
suggested re-wording.
Section 4:
We state that there is only one outcome of identity proofing here, but
in section one we list 4 outcomes. We should update and harmonize the
language in one of the two sections.
4.1.1.6 Address Confirmation
The term verified and validated seem to be used interchangeably with
regard to the address of record matching what is in records. I am not
sure if this is intentional or if one is preferred.
4.4.1.8 Security controls
Very confusing control. It says they shall have moderate or high and
then says they should have moderate.
Section 5
The table for identity evidence uses a wide array of terms for the
person being proofed – from applicant, to subject, to person, to
individual. I am not sure if this is intentional.

Left off at Table 5-2

jimfenton commented on sp800-63a/sec1_2_introduction.md:
```diff
@@ -6,17 +6,17 @@
 _This section is informative._
 
-This document provides requirements for enrollment and identity proofing of subscribers that wish to gain access to resources at each Identity Assurance Level (IAL).  The requirements detail the acceptability, validation, and verification of identity evidence that will be presented by an individual to support their claim of identity. This document also details the responsibilities of Credential S ...
+This document provides requirements for enrollment and identity proofing of subscribers that wish to gain access to resources at each Identity Assurance Level (IAL).  The requirements detail the acceptability, validation, and verification of identity evidence that will be presented by a subscriber to support their claim of identity. This document also details the responsibilities of Credential Se ...
```
At the point that they're presenting their evidence, they're an applicant rather than a subscriber.

jimfenton commented on sp800-63a/sec1_2_introduction.md:
```diff
@@ -37,23 +37,21 @@ The following table states which sections of the document are normative and whic
 **IAL3** - Physical presence is required for identity proofing. Identifying attributes must be verified by an authorized and trained representative of the CSP. As with IAL2, attributes could be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes.  A CSP that supports IAL3 can support IAL1 and IAL2 identity attributes if the user consents.
 
-At IAL2 and IAL3, pseudonymity in federated environments is enabled by limiting the number of attributes sent from the CSP to the RP, or the way they are presented. For example, if a RP needs a valid birthdate but no other personal details, the RP should leverage a CSP to request just the birthdate of the subscriber. It is preferred for the RP to ask the CSP for an attribute reference. For exampl ...
+At IAL2 and IAL3, pseudonymity in federated environments is enabled by limiting the number of attributes sent from the CSP to the RP, or the way they are presented. For example, if a RP needs a valid birthdate but no other personal details, the RP should leverage a CSP to request just the birthdate of the subscriber. It is preferred for the RP to ask the CSP for an attribute reference. For exampl ...
```
lower level --> lower-level
already verified --> already-verified
No comma after "noted", or perhaps just get rid of "It should be noted that"

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -8,17 +8,17 @@ _This section is normative._
 3. Home address
 
-It is permissible for the CSP to collect additional information in the process of identity proofing an applicant, provided validation and verification follow the requirements contained herein, and the applicant explicitly consents to the CSP collecting and storing the attributes.  
+The CSP may collect additional information in the process of identity proofing an applicant, provided validation and verification follow the requirements contained herein, and the applicant explicitly consents to the CSP collecting and storing the attributes.  
```
may --> MAY (normative word)

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
-	c. The applicant has been successfully proofed.
-	
+	a. The CSP matches the pictures on the license and the passport to the applicant's picture and determines they match.    
+	b. The CSP sends an enrollment code to the validated phone number of the applicant, the applicant provides the enrollment code to the CSP, and the CSP confirms they match, verifying the applicant is in possession and control of the validated phone number.  
```
verifying the applicant --> verifying that the applicant

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
+3. The CSP SHALL provide explicit notice at the time of collection to the applicant regarding the purpose for collecting and maintaining a record of the attributes necessary for identity proofing. This SHALL include whether such attributes are voluntary or mandatory in order to complete the identity proofing process and the consequences for not providing the attributes.
 
-4. The CSP SHALL NOT use attributes collected and maintained in the identity proofing process for any purpose other than identity proofing, authentication, or attribute assertions, or to comply with law or legal process unless the CSP provides clear notice and obtains consent from the subscriber for additional uses. CSPs SHALL NOT make consent a condition of the service.
+4. The CSP SHALL NOT use attributes collected and maintained for the identity proofing process for any purpose other than identity proofing, authentication, attribute assertions, or to comply with law or legal process unless the CSP provides clear notice and obtains consent from the applicant for additional uses. CSPs SHALL NOT make such consent a condition of the service.
```
attribute assertions --> attribute assertion

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
+5. The CSP SHALL provide effective mechanisms for redress of applicant complaints or problems arising from the identity proofing process. These mechanisms SHALL be easy for applicants to find and access.
 
-6. The identity proofing and enrollment processes SHALL be performed according to an applicable written policy or *practice statement* that specifies the particular steps taken to verify identities. The *practice statement* SHALL include control information that details how the CSP handles proofing errors that result in an applicant not being successfully enrolled.  For example, the number of ret ...
+6. The identity proofing and enrollment processes SHALL be performed according to an applicable written policy or *practice statement* that specifies the particular steps taken to verify identities. The *practice statement* SHALL include control information that details how the CSP handles proofing errors that result in an applicant not being successfully enrolled.  For example, the number of ret ...
```
Add comma after i.e.
counter-measures --> countermeasures

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
-	a. Any steps that it will take to verify the identity of the applicant beyond any mandatory requirements specified herein;  
-	b. The PII, including any biometrics, images, scans, or other copies of the identity evidence that the CSP will maintain as a record of identity proofing. Note: Specific federal, state, local, territorial, or tribal government and private sector organization requirements may apply; and
+	a. Any steps that it will take to verify the identity of the applicant beyond the mandatory requirements specified herein;  
+	b. The PII, including any biometrics, images, scans, or other copies of the identity evidence that the CSP will maintain as a record of identity proofing. Note: Specific Federal, state, local, territorial, or tribal government and private sector organization requirements may apply; and
```
AP style guide says federal is lowercase when used to distinguish from state, local, etc.

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
 8. All PII collected as part of the enrollment process SHALL be protected to ensure confidentiality, integrity, and attribution of the information source.
 
-9. The entire proofing transaction, including transactions that involve a third party, SHALL occur over an Authenticated Protected Channel.
+9. The entire identity proofing process, including transactions that involve a third party, SHALL occur over an Authenticated Protected Channel.
```
authenticated protected channel should not be capitalized.

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -32,75 +32,75 @@ The following provides a sample of how a CSP and an applicant interact during th
+9. The entire identity proofing process, including transactions that involve a third party, SHALL occur over an Authenticated Protected Channel.
 
-10. <a name="gr13"></a>The CSP SHOULD obtain additional confidence in identity proofing using fraud mitigation measures, for example inspecting geolocation, examining the device characteristics of the applicant, evaluating behavioral characteristics, or checking vital statistic repositories such as the [Death Master File](https://www.ssdmf.com/Library/InfoManage/Guide.asp?FolderID=1), so long as  ...
+10. <a name="gr13"></a>The CSP SHOULD obtain additional confidence in identity proofing using fraud mitigation measures, for example inspecting geolocation, examining the device characteristics of the applicant, evaluating behavioral characteristics, or checking vital statistic repositories such as the [Death Master File](https://www.ssdmf.com/Library/InfoManage/Guide.asp?FolderID=1), so long as  ...
```
comma after e.g. (second instance)

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -167,15 +167,15 @@ The CSP MAY collect biometrics for the purposes of non-repudiation and re-proofi
 #### 4.4.1.8. Security Controls
 
-The CSP SHALL employ appropriately tailored security controls, to include control enhancements, from the moderate or high baseline of security controls defined in [[SP 800-53]](#SP800-53) or equivalent federal (e.g. [[FEDRAMP]](#FEDRAMP))  or industry standard and SHOULD ensure that the minimum requirements associated with the *moderate* baseline are satisfied.
+The CSP SHALL employ appropriately tailored security controls, to include control enhancements, from the *moderate* or *high* baseline of security controls defined in [[SP 800-53]](#SP800-53) or equivalent Federal (e.g. [[FEDRAMP]](#FEDRAMP)) or industry standard and SHOULD ensure that the minimum requirements associated with the *moderate* baseline are satisfied.
```
lowercase federal, I believe. http://writingexplained.org/ap-style/ap-style-federal

appropriately tailored --> appropriately-tailored (this is issue #1527)

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -219,19 +219,19 @@ The CSP SHALL perform all identity proofing steps with the applicant in-person.
+#### 4.5.8. Security Controls
 
-The CSP SHALL employ appropriately tailored security controls, to include control enhancements, from the High baseline of security controls defined in [[SP 800-53]](#SP800-53) or an equivalent federal (e.g. [[FEDRAMP]](#FEDRAMP)) or industry standard and SHOULD ensure that the minimum requirements associated with the *high* baseline are satisfied.
+The CSP SHALL employ appropriately tailored security controls, to include control enhancements, from the *high* baseline of security controls defined in [[SP 800-53]](#SP800-53) or an equivalent Federal (e.g. [[FEDRAMP]](#FEDRAMP)) or industry standard and SHOULD ensure that the minimum requirements associated with the *high* baseline are satisfied.
```
federal
e.g.,

jimfenton reviewed (changes requested):
Good review; I have added my $0.02.

RGalluzzo committed:
Merge remote-tracking branch 'refs/remotes/usnistgov/post-public-comment' into post-public-comment

# Conflicts:
#	sp800-63a/sec4_ial.md
#	sp800-63a/sec5_proofing.md

RGalluzzo committed:
Merge remote-tracking branch 'refs/remotes/usnistgov/post-public-comment' into post-public-comment

# Conflicts:
#	sp800-63a/sec4_ial.md

RGalluzzo committed:
A copy edit review - batch #2 - 800-63A - section 5

comments:
Table 5-2
“confirming the integrity of physical security features and *lack of
fraudulent modification*.” This is an awkward statement. Suggest
re-wording or dropping the “lack of” phrase.  A change could be
“confirming the integrity of the physical security features and that the
document is not fraudulent or inappropriately modified.”
5.3.2
4. Should this read “KBV SHOULD be based on *the issuing source or*
multiple authoritative sources.”?
5. It seems difficult to move to from “should have multiple
authoritative sources” to “should use micro-deposits” which don’t really
leverage multiple authoritative sources. Perhaps flip 4 & 5 and go with
something like “Where transactional history is not used, KBV should be
based on multiple authoritative sources.”
6, a. Is this intended to read as if only one (1) of the questions needs
to be a free form answer?
5.3.3
Suggest changing “encounter” to “interaction” – encounter sounds
unnecessarily aggressive.
5.3.3.2
1. What qualifies as “departing?” Simply leaving field of view?
3. “The operator SHALL direct the applicant, as required, to remove any
doubt in the proofing process.” Doubt in what? – Suggest rephrasing to
“If the remote operator believes the applicant is attempting to
fraudulently proof by leveraging means out of their field of view or
that would otherwise compromise the validity of the identity proofing
session they SHALL ask the applicant to show their surroundings to
remove doubt or they SHALL immediately terminate the proofing session.”
5.3.4
4. Should “subscriber” actually be “trusted referee” here? Otherwise
it’s not clear why this would be here.

RGalluzzo commented:
It seems my commits from today were added into this pull request. Some comments in addition to the new commits are included below:
Table 5-2
 “confirming the integrity of physical security features and *lack of fraudulent modification*.” This is an awkward statement. Suggest re-wording or dropping the “lack of” phrase.  A change could be “confirming the integrity of the physical security features and that the document is not fraudulent or inappropriately modified.”
5.3.2
4. Should this read “KBV SHOULD be based on *the issuing source or* multiple authoritative sources.”?
5. It seems difficult to move to from “should have multiple authoritative sources” to “should use micro-deposits” which don’t really leverage multiple authoritative sources. Perhaps flip 4 & 5 and go with something like “Where transactional history is not used, KBV should be based on multiple authoritative sources.”
6, a. Is this intended to read as if only one (1) of the questions needs to be a free form answer? 
5.3.3
Suggest changing “encounter” to “interaction” – encounter sounds unnecessarily aggressive. 
5.3.3.2
1. What qualifies as “departing?” Simply leaving field of view?
3. “The operator SHALL direct the applicant, as required, to remove any doubt in the proofing process.” Doubt in what? – Suggest rephrasing to “If the remote operator believes the applicant is attempting to fraudulently proof by leveraging means out of their field of view or that would otherwise compromise the validity of the identity proofing session they SHALL ask the applicant to show their surroundings to remove doubt or they SHALL immediately terminate the proofing session.”
5.3.4
4. Should “subscriber” actually be “trusted referee” here? Otherwise it’s not clear why this would be here.

RGalluzzo committed:
A- Copy edit review, Final Batch

RGalluzzo commented:
Yep. Messed that up! Either way, this is the final set of commits for this PR. Comments from the last review are below. Let me know if you have questions. 
Section 6
Use case #2. This just seems to be saying that “you want to get a credential.” I am not sure why this is considered “deriving.” It would seem like this should read something like: 
An applicant seeks to either add a new authenticator to an existing identity record or establish a new credential at a new CSP leveraging a previously created identity record. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g., basic browsing vs. financial). *This use case is covered by allowable identity evidence in [Section 5.2](#validate).*
Section 7
“The threats to the enrollment process include impersonation attacks and threats to the transport mechanisms for identity proofing, authenticator binding, and credential issuance. [Table 7-1](#63aSec7-Table1) lists the threats related to enrollment and identity proofing.” This seems to suggest there are more than just the two categories in the previous section. Are there actually three categories of threats: 1) Impersonation, 2) Intercept (mitm? Or something similar), 3) malfeasance of the infrastructure provider?
Denial of enrollment also doesn’t seem to fit neatly into impersonation either. Maybe the statement regarding categories is unnecessary?
Removed repetition of “during issuance” from the “issuance” attacks. Seemed redundant. Also—with these it seems like the examples overlap with MITM and “Duplication” from B. Is the new name for these solely because they are being placed in proofing? Or, do the examples need to be updated. For example:
Disclosure – an individual’s PII being submitted for proofing is collected by an attacker positioning themselves between the CSP and the applicant.
Tampering- An attacker modifies applicant information being submitted to the CSP during registration by positioning themselves between the CSP and applicant. 
Mitigation tables:
I am not sure I see much of a difference between the two social engineering examples.
Why combine Unauthorized issuance/social engineering, but not disclosure and tampering?
Section 8
8.1- I would suggest making reference to the benefits of reducing RP risk when discussing data minimization. The benefits as described now are somewhat vague. I am not sure how data minimization equates to more trust in the identity proofing process. 
8.1.1-“ Federal agencies need to review any decision to collect the SSN relative to their obligation to reduce the collection and unnecessary use of SSNs under Office of Management and Budget policy.” should probably quote the policy here. 
8.6- the use of authentication in this section seems inconsistent with the concept of a “digital identity guideline” rather than “electronic authentication guidelines.”

jimfenton commented:
Replaced by #1805
